estuary/models/distgit.py

```diff
@@ -33,6 +33,7 @@ class DistGitBranch(EstuaryStructuredNode):
 class DistGitCommit(EstuaryStructuredNode):
     """Definition of a dist-git commit in Neo4j."""
 
+    __label_display__ = 'commit'
```

Comment:
The label display in bugzilla bug, freshmaker event, koji build starts with a capital letter while commit, advisory start with lower case. Could you please make it uniform

Reply:
@sarah256 and I had a chat on IRC and she explained me the reason behind it. So I think it's fine as it is

Reply:
@sarah256 and @yashvardhannanavati we could consider title-casing it and then for @sarah256's purposes she could just call `.lower()` on the string. I'll leave this up to you both to decide.